python/ray/experimental/data/impl/compute.py

```diff
@@ -27,19 +43,13 @@ def apply(self, fn: Any, remote_args: dict,
         kwargs = remote_args.copy()
         kwargs["num_returns"] = 2
 
-        @ray.remote(**kwargs)
-        def wrapped_fn(block: Block, meta: BlockMetadata):
-            new_block = fn(block)
-            accessor = BlockAccessor.for_block(new_block)
-            new_meta = BlockMetadata(
-                num_rows=accessor.num_rows(),
-                size_bytes=accessor.size_bytes(),
-                schema=accessor.schema(),
-                input_files=meta.input_files)
-            return new_block, new_meta
+        # Lazy init to avoid circular import.
+        global _remote_fn
```

Comment:
TODO: we should move all remote function defs into a dedicated file to avoid re-definition / import issues.